ci_tests/cfg-indent-check.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
```

Comment:
What do you think about making this script to be a general linter for the cartesian cfg files? By doing so we can add additional rules easily upon new requirements.

Reply:
So do you mean to maintain a separate file or copy from VT? Currently we don't have a valid linter so I just made basic checks in this script.